backend/entities/session.entity.ts

```diff
@@ -0,0 +1,28 @@
+import { Entity, Column, PrimaryGeneratedColumn, ManyToOne, OneToMany } from 'typeorm';
+import { User } from './user.entity';
+import { Participant } from './participant.entity';
+import { Game } from './game.entity';
+
+@Entity()
+export class Session {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  date: Date;
+
+  @Column()
+  url: string;
+
+  @Column()
+  isActive: boolean;
+
+  @ManyToOne(() => User, user => user.sessions)
+  user: User;
+
+  @OneToMany(() => Participant, participant => participant.session)
+  participants: Participant[];
+
+  @OneToMany(() => Game, game => game.session)
+  games: Game[];
```

Comment:
why oneToMany relation?

a user have many sessions and each session itself have many games?

Reply:
yes. So lets say there is sprint 28 i.e., 1 session started by scrum master and then they play 8 games for 8 tickets

Reply:
at the same time how can a scrum master have different games running its not a case.

for every session there will be one game?

Reply:
or else we can discuss in a call